package.json

```diff
@@ -22,7 +22,7 @@
     "generate": "run-s generate:*",
     "generate:imports": "node tools/generate-imports.mjs",
     "git-check": "node tools/check-git-version.mjs",
-    "jest": "cross-env LOG_LEVEL=fatal jest --logHeapUsage",
+    "jest": "cross-env LOG_LEVEL=fatal GIT_ALLOW_PROTOCOL=file jest --logHeapUsage",
```

Comment:
where did you found that variable?

Reply:
I forget exactly where

Reply:
@viceice and @rarkins I think this is the documentation for the `GIT_ALLOW_PROTOCOL` variable:

- https://git-scm.com/docs/git#Documentation/git.txt-codeGITALLOWPROTOCOLcode
- https://git-scm.com/docs/git-config#Documentation/git-config.txt-protocolallow